test/emoji_test.rb

```diff
@@ -38,11 +38,6 @@ class EmojiTest < TestCase
     assert_equal ["\u{0031}\u{fe0f}\u{20e3}", "\u{0031}\u{20e3}\u{fe0f}"], emoji.unicode_aliases
   end
 
-  test "unicode_aliases doesn't necessarily include form without VARIATION_SELECTOR_16" do
-    emoji = Emoji.find_by_unicode("\u{00a9}\u{fe0f}")
-    assert_equal ["\u{00a9}\u{fe0f}"], emoji.unicode_aliases
```

Comment:
How come this test is going away?

Reply:
The Apple palette included emoji versions of the `TEXT_GLYPHS` unicode characters but the Unicode palette has the unicode versions so that when we add the unicode aliases [here](https://github.com/github/gemoji/pull/155/files#diff-6bb18b552442b33796092b3d119d0633R106) we now have the emoji with the variation selector and without it.

While I was writing out this comment I tested this out and think I figured out why this test and functionality was here in the first place. The Unicode versions of the emojis look pretty bad 😄 

![Screen Recording 2019-05-01 at 16 02 42](https://user-images.githubusercontent.com/318208/57023758-c91c1d80-6c2a-11e9-9379-fb914a67cb1b.gif)

I'll fix this and add the test back in.

Reply:
Actually not sure why the `TEXT_GLYPHS` are in here now since the GIF above is actually how github.com works right now as well 😄